modules/atomic-widgets/elements/has-atomic-base.php

```diff
@@ -141,6 +141,7 @@ final public function get_raw_data( $with_html_content = false ) {
 		$raw_data = parent::get_raw_data( $with_html_content );
 
 		$raw_data['styles'] = $this->styles;
+		$raw_data['isAtomic'] = true;
```

Comment:
@nevo-elementor @StyleShit Can you please review this for me?

This approach works, but I would appreciate to hear your thoughts about this as well.